beacon_node/http_api/src/aggregate_attestation.rs

```diff
@@ -18,13 +18,14 @@ pub fn get_aggregate_attestation<T: BeaconChainTypes>(
     endpoint_version: EndpointVersion,
     chain: Arc<BeaconChain<T>>,
 ) -> Result<Response<Body>, warp::reject::Rejection> {
-    if endpoint_version == V2 {
+    let fork_name = chain.spec.fork_name_at_slot::<T::EthSpec>(slot);
```

Comment:
The spec requires a `Eth-Consensus-Version` header for V2. 
Wondering if we are supposed to derive the fork name from there instead of here.
The spec isn't very clear on that imo

Reply:
I think deriving it from the slot and ignoring the header is fine. If they're inconsistent it will be a result of misconfiguration. We could add a `WARN` log if this is detected, but I think this misconfiguration would be detected by the VC separately anyway (in `is_compatible`).